include/invert_quda.h

```diff
@@ -1139,16 +1236,22 @@ namespace quda {
 
     EigCGArgs *eigcg_args;
 
-    TimeProfile &profile;    //time profile for initCG solver
+    TimeProfile &profile; // time profile for initCG solver
 
     bool init;
 
-  public:
-
+public:
     IncEigCG(DiracMatrix &mat, DiracMatrix &matSloppy, DiracMatrix &matPrecon, SolverParam &param, TimeProfile &profile);
 
     virtual ~IncEigCG();
 
+    /**
+       @brief Expands deflation space.
+       @param V container of new eigenvectors
```

Comment:
How is V a container? Is it using a Composite field?

Reply:
Yes, this is a composite field.  Made this clear in my next push.